CHANGES.rst

```diff
@@ -4,18 +4,18 @@ Changelog
 11.2 (unreleased)
 -----------------
 
-Bug fixes
-+++++++++
+- Nothing changed yet.
+
+
+11.1.2 (2023-03-09)
+-------------------
 
 - Execute teardown when test was skipped in setup phase of a fixture.
 
 
 11.1.1 (2023-02-17)
 -------------------
 
-Bug fixes
-+++++++++
-
```

Comment:
I would have likes to have this heading even for an explicit bug fix release. Maybe next time.

Reply:
Added them back.